getting-started/singleframe.rst

```diff
@@ -0,0 +1,164 @@
+
+.. code-block:: bash
+
+   pipetask run -b $GEN3_RC2_SUBSET_DIR/SMALL_HSC/butler.yaml -p $GEN3_RC2_SUBSET_DIR/pipelines/DRP.yaml#singleFrame -i HSC/RC2/defaults -o u/$USER/single_frame --register-dataset-types
+
+There are many arguments to command:``pipetask run``.
+You can get useful information by saying command:``pipetask run --help``, but let's go over the ones listed here.
+
+The ``-b`` option specifies which butler definition to use when constructing the ``Butler`` object to use in processing.
+
+The ``-p`` option specifies which pipeline to run.
+The full pipeline definition lives in the ``DRP.yaml`` file, but subtasks of the full processing can be run by specifying the subtask name with the ``#`` character, e.g. ``#singleFrame`` in this case.
+
+The ``-i`` option indicates the input collections to use in processing.
+You will learn more about collections later in this document.
+
+The ``-o`` option defines the output collection to send the results of the processing to.
+These tutorials suggest that you put the outputs in collections under a namespace defined by your username since that is unique for a given system.
+In this case, there is little reason to be so careful because you are likely to have cloned into a space shared with others.
+However, it is good practice for times when you may be using a repository with a registry used by other users on the same system.
+
+The ``--register-dataset-types`` switch tells the butler to register a dataset type if it doesn't already have a definition for it.
+Because pipelines are allowed to define datasets at runtime, this switch is necessary if you expect products to be produced that are not already represented in the registry as in this case where we are producing calibrated exposures in a repository that contains only ``raw`` files.
+If you expect that all of the dataset types should already be registiered, as is the case when processing another subset of data with a pipeline that has already been run, it can help catch unexpected behavior to remove that switch.
+
+.. tip::
+   It is not included in the above command, but the ``-j`` option is useful if you have more than one core available to you.
+   Specifying ``-j #`` will run in parallel where ``#`` is the number of processes to execute in parallel.
+
+   `Dataset queries`_ can be specified using the ``-d`` argument to specify which specific datasets should be considered when building the execution graph.
+   If this argument is omitted, all data in the repository that can be processed based on other inputs, e.g. calibrations, will be.
+
+Aside: collections and quantum graphs
+============================================
+
+`Collections`_ are the primary way data in butler repositories are organized.
+Of the types of collections available, the two of interest here are the ``RUN`` and ``CHAINED`` types.
+
+``RUN`` collections are the least flexible.
+Once a dataset is added to a ``RUN`` collection, it can never be removed.
```

Comment:
Yes it can be removed. See `butler prune-datasets`. The point is that every single dataset put into a butler is allowed to exist solely in one RUN collection. It can't be moved later. You can make it appear somewhere else by adding it to a TAGGED collection.